Update UI to align with the "Superchain" narrative (#215)
LiskHQ/lisk-docs: configuration.md

```

To verify all you have properly configured your node, open the web client using `https://MY_IP_OR_HOST`. You should now see a secure SSL connection.

## Logging 

> For monitoring or debugging your node, Lisk Core tracks all important events that happen on the node by creating log messages for them.

Reviewer: ```suggestion
For monitoring or debugging your node, Lisk Core tracks all activity that happen in the node by creating log messages for them.
```
